src/main/resources/application.conf

```diff
@@ -518,7 +504,7 @@ mantis {
       block-cache-size = 33554432
     }
 
-    # Define which database to use [leveldb, rocksdb], iodb is not currently used
+    # Define which database to use [rocksdb], iodb is not currently used
```

Comment:
Do we need that anymore?

Reply:
probably not, but I would leave it as it is for now (as it is not critical). We can improve it later, when we will have working sync.